actionpack/lib/action_controller/railtie.rb

```diff
@@ -67,5 +67,12 @@ class Railtie < Rails::Railtie #:nodoc:
         config.compile_methods! if config.respond_to?(:compile_methods!)
       end
     end
+
+    initializer "setup session store" do |app|
+      ActiveSupport.on_load(:action_controller) do
```

Comment:
This is just masking a problem. It only works because ActionController::Base is not loaded in those tests.